source/Hardpoint.cpp

```diff
@@ -223,14 +223,18 @@ void Hardpoint::Fire(Ship &ship, vector<Projectile> &projectiles, vector<Visual>
 	aim += angle;
 	start += aim.Rotate(outfit->HardpointOffset());
 
+	// Determine the inaccuracy to add to a new projectile
+	double outfitInaccuracy = outfit->Inaccuracy();
+	Angle projectileInaccuracy = Angle::Random(outfitInaccuracy) - Angle::Random(outfitInaccuracy);
```

Comment:
You mean this?
```suggestion
	// Determine the inaccuracy to add to a new projectile
	Angle projectileInaccuracy;
	if(outfitInaccuracy)
		projectileInaccuracy = Angle::Random(outfitInaccuracy) - Angle::Random(outfitInaccuracy);

Reply:
I pushed this fix in the 2 locations it's used.

Reply:
There's no need to have the additional angle - `aim` is _always_ combined with it every time it is used. So update `aim` once rather than create three separate temporaries.

Reply:
Oh.  I'll see if that applies to submunition launching too.

Reply:
I kept the 1 temporary to avoid calling the function more than once.